00-Introducing-HOOMD-blue/06-Equilibrating-the-System.ipynb

```diff
@@ -32,6 +32,20 @@
     "import math"
```

Comment:
<p>I think we can reasonably assume a hoomd user knows how to remove files. Is that the reason this was added?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>This is to ensure that we don't get an exception when the file is created in <code>xb</code>  mode.</p>

Reply:
<p>This doesn't work for me in a test run. The file is not removed.</p><p>I launched jupyter lab  version 2.2.9 in the directory hoomd-examples/  and then navigated to this notbook and ran it. The current working directory is <code>hoomd-examples/00-Introducing-HOOMD-blue</code>.</p><p> </p><p>I get similar behavior with <code>jupyter notebook</code></p>